aten/src/ATen/core/jit_type.h

```diff
@@ -1100,19 +1100,19 @@ CAFFE2_API TypePtr evalTypeVariables(TypePtr type, TypeEnv & type_env);
 
 struct ClassType;
 using ClassTypePtr = std::shared_ptr<ClassType>;
-using ::torch::jit::script::Module;
-using ::torch::jit::script::Method;
+using ::torch::jit::script::CompilationUnit;
+using ::torch::jit::script::Function;
 
 // This represents a class in TorchScript.
 struct CAFFE2_API ClassType : public Type {
   // Create a user type and register it globally.
   static ClassTypePtr create(
       const std::string& name,
-      std::shared_ptr<Module> module);
+      std::shared_ptr<CompilationUnit> module);
```

Comment:
I've fixed this one, but there are a number of places in this PR where variables names were like:` `Method method`. I didn't update them all because of the effort required to do that compared to the value. We should nit-pick reviews to give better names to things `TypeName type_name;` is normally a pretty lazy name, but I've done it a bunch of times!